package.json

```diff
@@ -9,7 +9,12 @@
     "lint": "eslint pages src",
     "lint:errors": "eslint pages src --quiet",
     "cypress:open": "cypress open",
-    "cypress:run": "cypress run"
+    "cypress:run": "cypress run",
+    "prepare": "husky install"
```

Comment:
I know we do not deploy planet-donations anywhere else but with Vercel, but do you want to make the same change as for planet-webapp to support other deployments in the future disabling this check for production releases? https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/package.json#L24

Reply:
Done